docs/source/llms/tracing/index.rst

```diff
@@ -467,6 +467,17 @@ filter strings, and other parameters.
         experiment_ids=["1", "2"], filter_string="attributes.status = 'OK'", max_results=5
     )
 
+Alternatively, you can use `fluent API <./index.html#tracing-fluent-apis>`_ to search for traces. This method allows you to specify fields to extract from traces using the format 
+``"span_name.[inputs|outputs]"`` or ``"span_name.[inputs|outputs].field_name"``.
```

Comment:
Can we also explain:

1. The extracted fields are included as extra columns in the pandas dataframe returned by `search_traces())
2. This feature can be used to build evaluation datasets to further improve model / agent performance